### PR TITLE
Change the example script from ruby to elixir

### PR DESCRIPTION
**Pull request by pasja**

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@pasja Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/rabbitmq?repositoryId=rabbitmq/rabbitmq-website&pullRequestId=498)!

[Click here](https://cla.pivotal.io/sync/rabbitmq?repositoryId=rabbitmq/rabbitmq-website&pullRequestId=498) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by michaelklishin** [comment]
Doh, thank you!

File: site/tutorials/tutorial-one-elixir.md [hunk 1]
```diff
@@ -206,7 +206,7 @@ run before. But we're not yet sure which
 program to run first. In such cases it's a good practice to repeat
 declaring the queue in both programs.
 
-[Here's the whole send.rb script](https://github.com/rabbitmq/rabbitmq-tutorials/blob/master/ruby/send.rb).
+[Here's the whole send.esx script](https://github.com/rabbitmq/rabbitmq-tutorials/blob/master/elixir/send.exs).
```
Comment (lukebakken) [inline site/tutorials/tutorial-one-elixir.md]: PS - I changed `esx` to `exs` here - 7821b24339d1e8f04ac1ade50925a6d75fe3799d

**Comment by pasja** [comment]
Thanks!

It seems that a typo correction can have its own typos :-)

**Pull request merged**